relay-server/src/actors/envelopes.rs

```diff
@@ -1006,6 +1006,19 @@ impl EnvelopeProcessor {
         });
     }
 
+    fn process_replay_recordings(&self, state: &mut ProcessEnvelopeState) {
+        let replays_enabled = state.project_state.has_feature(Feature::Replays);
+        state.envelope.retain_items(|item| match item.ty() {
+            ItemType::ReplayRecording => {
+                if !replays_enabled {
+                    return false;
+                }
+                true
+            }
```

Comment:
_nit_: Can be shortened to:

```suggestion
            ItemType::ReplayRecording => replays_enabled,
```